packages/store/src/keys/compareKeys.js

```diff
@@ -116,7 +119,8 @@ export const compareKeys = (left, right) => {
       switch (leftTag) {
         case 'copySet': {
           // copySet X is smaller than copySet Y when every element of X
-          // is keyEQ to some element of Y.
+          // is keyEQ to some element of Y and Y has at least one element
+          // that no element of X is keyEQ to.
```

Comment:
Note the strengthening here. If every element of X is compares equal to an element of Y and Y does not have an element for which no element of X compares equal, then they have the same elements and X would compare equal to Y rather than smaller.